src/component/SettingsModal/index.jsx

```diff
@@ -24,8 +22,8 @@ SettingsModal.propTypes = {
 };
 
 SettingsModal.defaultProps = {
-  toggleShow: 'toggleShow',
-  show: 'show',
+  toggleShow: Function,
+  show: Boolean,
```

Comment:
`Function` and `Boolean` is a class/function, the default value should be an instance/value. So `show` should be either `true` or `false`, and `toggleShow` should either be an empty function, or you make it required.

In my opinion, it would be better to just make it required because the behavior of the modal would break if you don't have a function that toggles its appearance on the screen.